Tribler/Core/defaults.py

```diff
@@ -147,6 +148,11 @@
 sessdefaults['watch_folder']['enabled'] = False
 sessdefaults['watch_folder']['watch_folder_dir'] = None
 
+# HTTP API config
+sessdefaults['http_api'] = OrderedDict()
+sessdefaults['http_api']['enabled'] = True
+sessdefaults['http_api']['port'] = 8081
```

Comment:
We can't have hardcoded ports on the config file, it breaks multiple Tribler instances running on the same machine. (Used in experiments, tests and such).